app/src/app/map/Map.module.scss

```diff
@@ -0,0 +1,14 @@
+@import "../../theme/base";
+
+.map-container {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  margin-top: -$navbar-height + $space-default;
```

Comment:
Why are we pulling this up?

Reply:
Ah ya vi. este es el MapView de Sala de Inversión

Yo creo que hay que editar mejor el `@mixin withNavbar` y bajarle el margin general.